pkg/test/runner/runner.go

```diff
@@ -113,15 +112,11 @@ func (r *Runner) runFnEval() error {
 	}
 	if r.testCase.Config.EvalConfig.Image != "" {
 		kptArgs = append(kptArgs, "--image", r.testCase.Config.EvalConfig.Image)
-	} else if r.testCase.Config.EvalConfig.ExecPath != "" {
-		kptArgs = append(kptArgs, "--exec-path", r.testCase.Config.EvalConfig.ExecPath)
+	} else if r.testCase.Config.EvalConfig.ExecUniquePath.String() != "" {
```

Comment:
`.String()` method is for display etc. `string(...ExecUniquePath)` will be the right usage here.

I realized, `UniquePath` should just have a method name `IsEmpty()`.